app/Models/Voucher.php

```
@@ -146,44 +144,49 @@ public function completeRequestStatus()
         $this->request_status = RequestStatuses::APPROVED->value;
         $this->save();
         $this->refresh();
+        $journalEntry = $this->journalEntry()->with('paymentRequest')->first();
+        $paymentRequestId = $journalEntry->paymentRequest->id;
         if ($this->type === VoucherType::DISBURSEMENT->value) {
             TransactionFlowService::updateTransactionFlow(
-                $this->id,
+                $paymentRequestId,
                 TransactionFlowName::DISBURSEMENT_VOUCHER_APPROVAL->value,
                 TransactionFlowStatus::DONE->value
             );
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Services\TransactionFlowService' in method 'completeRequestStatus'. (undefined)

(StaticAccess)